src/app/store/entities/sandboxes/boilerplates/default-boilerplates.js

```diff
@@ -1,6 +1,7 @@
 export const JS = {
   id: 'js',
-  condition: '\.js$', // eslint-disable-line
+  extension: '.js',
+  condition: '.js$', // eslint-disable-line
```

Comment:
Why? As a general rule, IMO, you should add the eslint rule that you are disabling on the line. Ex:
```js
// eslint-disable-line no-use-before-define
```
otherwise you could be masking a different eslint error